packages/core/src/app/utils/range.ts

```diff
@@ -66,7 +66,11 @@ export const isRangeSubset = (subset: Range, superset: Range): boolean => {
 }
 
 /**
- * Determines whether the provided Ranges collectively span the Range in question
+ * Determines whether the provided Ranges collectively span the Range in question.
+ * For instance,
+ * doRangesSpanRange([{start: 1, end: 3}, {start: 3,  end: 5}], {start: 2, end: 4})
```

Comment:
Actually, adding the 'do' makes more sense! The function name needs to be updated as well to correspond with the function name in the comment :)

Reply:
I like the comments that were added, definitely a lot easier to understand, thanks!